Dockerfile

```diff
@@ -0,0 +1,96 @@
+FROM        php:7.1-apache
+
+######################################
+# Install and configure dependencies #
+######################################
+
+RUN         apt-get update \
+            && apt-get upgrade -y \
+            && apt-get install -y \
+                bzip2 \
+                curl \
+                fontconfig \
+                g++ \
+                git \
+                gnupg \
+                libfreetype6 \
+                libicu-dev \
+                libjpeg-dev \
+                libpcre3-dev \
+                libpng-dev \
+                libzip-dev \
+                mysql-client \
+                postgresql-9.4 \
+                postgresql-server-dev-all \
+                unzip \
+                zip \
+            && rm -rf /var/lib/apt/lists/* \
+            && docker-php-ext-install exif \
+            && docker-php-ext-configure gd --with-jpeg-dir=/usr/ \
+            && docker-php-ext-install gd \
+            && docker-php-ext-install intl \
+            && docker-php-ext-install mbstring \
+            && docker-php-ext-install pdo_mysql \
+            && docker-php-ext-install pdo_pgsql \
+            && docker-php-ext-configure zip --with-libzip \
+            && docker-php-ext-install zip \
+            && a2enmod rewrite \
+
+
+####################
+# Install Composer #
+####################
+
+            && curl -sS https://getcomposer.org/installer | php \
+            && mv composer.phar /usr/local/bin/composer \
+            && mkdir -p /var/www/.composer/vendor/ \
+
+###############
+# Install NPM #
+###############
+
+            && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+            && apt install -y nodejs \
+
+################################
+# Install Grunt, Gulp, Webpack #
```

Comment:
What about Robo task runner? Shouldn't it be here?

Reply:
Nope as we have decided internally, Robo will no be installed globally anymore (Drush as well).

Reply:
@fousjiri  no, we decided to have it as a project dependency defined in `composer.json`